3rdparty/optee/libutee/CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+set(LIBUTEE_S_SRC
+  ${OE_TZ_LIBUTEE_SRC}/arch/arm/utee_syscalls_a64.S
```

Comment:
Where's `OE_TZ_LIBUTEE_SRC` set and whey do we need to reference it over and over, can we just use a short relative path maybe?

Reply:
(Based on where I found it, looks like this _could_ just be `optee/os` here.)

Reply:
```
OE_TZ_OPTEE_SRC = ${PROJECT_SOURCE_DIR}/3rdparty/optee/optee_os
OE_TZ_LIBUTEE_SRC = ${OE_TZ_OPTEE_SRC}/lib/libutee

=> OE_TZ_LIBUTEE_SRC = <repo root>/3rdparty/optee/optee_os/lib/libutee
```
I followed the pattern here [this](https://github.com/microsoft/openenclave/blob/master/libc/CMakeLists.txt#L79).

Reply:
Ah that makes sense. That pattern is the way it is because the CMakeLists.txt is in `src/libc` while the the 3rdparty code is `../../3rdparty/musl/musl`.

Since this file exists at `3rdparty/optee/libutee/CMakeLists.txt` and the code exists at `3rdparty/optee/optee_os/lib/libutee`...you're fine. Looking at those two paths you'd have to `../optee_os/lib/libutee` and that's just as annoying.